src/qt/bitcoingui.cpp

```diff
@@ -720,6 +725,14 @@ void BitcoinGUI::optionsClicked()
     dlg.exec();
 }
 
+void BitcoinGUI::openConfigClicked()
+{
+    boost::filesystem::path pathConfig = GetDataDir() / "gridcoinresearch.conf";
+    /* Open gridcoinresearch.conf with the associated application */
+    if (boost::filesystem::exists(pathConfig))
```

Comment:
Should probably add an error message if it doesn't exist so that there isn't a case where the button just doesn't do anything.

Reply:
I'm not sure that's necessary. At startup a new config file will be created if necessary. And I don't think users will delete it while the wallet is running. Maybe @jamescowens or @cyrossignol can comment on this.

Reply:
Init will create a config file if it doesn't exist. I don't think we need to display a message if the user removes it, but it may be worth doing the enhancement later. 

@Pythonix Please use `GetConfigFile()` instead of hardcoding the filename so that this will work when a user specifies the config file via the `-conf` option.